src/components/PreJoinScreens/PreJoinScreens.tsx

```diff
@@ -53,15 +50,8 @@ export default function PreJoinScreens() {
     setStep(Steps.deviceSelectionStep);
   };
 
-  const SubContent = (
-    <>
-      {Video.testPreflight && <PreflightTest />}
-      <MediaErrorSnackbar error={mediaError} />
-    </>
-  );
-
   return (
-    <IntroContainer subContent={step === Steps.deviceSelectionStep && SubContent}>
+    <IntroContainer error={mediaError}>
       {step === Steps.roomNameStep && (
         <RoomNameScreen
           name={name}
```

Comment:
Here, I think it might be a little simpler if we just had the `<MediaErrorSnackbar />` as a child of the IntroContainer, instead of a prop:

```
  return (
    <IntroContainer>
      <MediaErrorSnackbar error={mediaError} />

      {step === Steps.roomNameStep && (
        <RoomNameScreen
          name={name}
          roomName={roomName}
          setName={setName}
          setRoomName={setRoomName}
          handleSubmit={handleSubmit}
        />
      )}

      {step === Steps.deviceSelectionStep && (
        <DeviceSelectionScreen name={name} roomName={roomName} setStep={setStep} />
      )}
    </IntroContainer>
  );
```